corehq/apps/export/models/new.py

```diff
@@ -692,7 +690,7 @@ def _get_sub_documents_helper(document_id, path, row_docs):
             else:
                 next_doc = {}
             if path[0].is_repeat:
-                if type(next_doc) != list:
+                if next_doc is not list:
```

Comment:
This does something different than what it did before. I think it should use `isinstance()`.
```suggestion
                if not isinstance(next_doc, list):
```

Reply:
My bad. I'm wondering if I'm still running an out of date version of flake8, because the error instructions were:
```
do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance`
```

and yet the error code was [E721](https://www.flake8rules.com/rules/E721.html), which only includes the line about `isinstance` now. I kind of out-thought myself and assumed the code explicitly didn't want to check for subclasses, but I still made the mistake. In the rare case that that was true for this code, I think the correct line would be:
```
if type(next_doc) is list:
```
right? Regardless, it seems likely that this code wouldn't be disrupted by subclasses